patterns/left-aligned-cta-image.php

```diff
@@ -6,26 +6,16 @@
  */
 ?>
 
-<!-- wp:group {"align":"wide","layout":{"type":"constrained","wideSize":"","contentSize":"1280px"}} -->
-<div class="wp-block-group alignwide"><!-- wp:columns {"align":"full"} -->
-<div class="wp-block-columns alignfull"><!-- wp:column {"width":"642px","style":{"spacing":{"padding":{"top":"40px","right":"40px","bottom":"40px","left":"40px"}}}} -->
+<!-- wp:group {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"constrained","wideSize":"","contentSize":"1280px"}} -->
+<div class="wp-block-group alignwide" style="margin-top:var(--wp--preset--spacing--50);margin-bottom:var(--wp--preset--spacing--50)"><!-- wp:columns {"align":"full"} -->
+<div class="wp-block-columns alignfull"><!-- wp:column {"style":{"spacing":{"padding":{"top":"40px","right":"40px","bottom":"40px","left":"0px"}}}} -->
 <div class="wp-block-column" style="padding-top:40px;padding-right:40px;padding-bottom:40px;padding-left:40px;flex-basis:642px"><!-- wp:heading -->
```

Comment:
When I add the pattern I am getting a block validation error, because the padding-left and the flex bases are incorrect:

```

logger.js:17 Block validation: Block validation failed for `core/column` (Object).

Content generated by `save` function:

<div class="wp-block-column" style="padding-top:40px;padding-right:40px;padding-bottom:40px;padding-left:0px"></div>

Content retrieved from post body:

<div class="wp-block-column" style="padding-top:40px;padding-right:40px;padding-bottom:40px;padding-left:40px;flex-basis:642px">



</div>
(anonymous) @ logger.js:17
```

Reply:
Thanks for looking into it Carolina 😄 - I adjusted things to fix the issue.